src/controller/controller.c

```diff
@@ -397,6 +397,35 @@ bool controller_apply_config(Controller *controller) {
                 }
         }
 
+        _cleanup_freev_ char **sections = cfg_list_sections(controller->config);
+        if (sections == NULL) {
+                bc_log_error("Failed to list config sections");
+                return false;
+        }
+        for (size_t i = 0; sections[i] != NULL; i++) {
+                const char *section = sections[i];
+                if (!str_has_prefix(section, CFG_SECT_NODE_PREFIX)) {
+                        continue;
+                }
+                const char *node_name = section + strlen(CFG_SECT_NODE_PREFIX);
+                Node *node = controller_find_node(controller, node_name);
+                if (node == NULL) {
+                        /* Add it to the list if it is marked allowed */
+                        bool allowed = cfg_s_get_bool_value(controller->config, section, CFG_ALLOWED);
+                        if (allowed) {
+                                node = controller_add_node(controller, node_name);
+                        } else {
+                                continue;
+                        }
```

Comment:
NIT
```suggestion
                        if (allowed) {
                                node = controller_add_node(controller, node_name);
                        }
```

Reply:
Not sure what you mean. We need the continue, because if node is not on the allowed name list, and is not Allowed=true, then node will be NULL, and we will try to apply e.g. the selinux context option on a NULL node below.